backend/src/agents/crew.py

```
@@ -0,0 +1,400 @@
+import json
+import yaml
+from crewai.project import CrewBase, agent, crew, task
+import logging
+from tools.browser_tools import GoToPageTool,FetchAndCleanHTMLTool, GoBackTool, ReloadPageTool, GetCurrentURL, HoverElementTool, SelectDropdownTool, ScrollPageTool, DoubleClickTool, TextDeleteTool, TakeScreenshotTool, ClickElementTool, FillInputTool
+from utils.browser_manager import BrowserManager
+from playwright.async_api import Page
+from typing import List, Optional, Literal
+from enum import Enum
+from typing import Union
+
+from utils.browser_manager import BrowserManager 
```

Ruff (0.12.2)

Redefinition of unused `BrowserManager` from line 9

Remove definition: `BrowserManager`

(F811)